src/rgw/rgw_admin.cc

```diff
@@ -4238,6 +4242,8 @@ int main(int argc, const char **argv)
       cerr << "couldn't init config storage provider" << std::endl;
       return EIO;
     }
+    JSONFormattable sal_config;
+    sal_config.set("type", "rados");
```

Comment:
This isn't going to work, unless it's just a placeholder for now.  Currently, radosgw-admin get's it's config from ceph.conf, and will use the correct store.  This allows you to create users for DBStore, for example.  So going forward, radosgw-admin needs to load the json like the main RGW does, and use the correct store.